kratos/tests/cpp_tests/geometries/test_coupling_geometry.cpp

```diff
@@ -95,6 +109,59 @@ namespace Kratos {
             }
         }
 
+        /**
+         * Test for checking using with elements
+         * Test 2
+         */
+        KRATOS_TEST_CASE_IN_SUITE(CouplingElements, KratosCoreGeometriesFastSuite) {
+            auto p_triangle_master = GenerateNodeTriangle();
+            auto p_triangle_slave = GenerateNodeTriangle2();
+
+            Geometry<NodeType>::Pointer p_coupling_geometry = Kratos::make_shared<CouplingGeometry<NodeType>>(p_triangle_master, p_triangle_slave);
+
+            // Creating a model and an element
+            Model current_model;
+            ModelPart& r_model_part = current_model.CreateModelPart("Main");
+            auto p_elem_prop = r_model_part.CreateNewProperties(1);
+            auto p_element = r_model_part.CreateNewElement("Element2D3N", 1, p_triangle_master->Points(), p_elem_prop); // Creating the element
```

Comment:
Should be possible to directly put the geometry. This comes with #4872

Reply:
The CreateNewElement should allow pointer to geometries directly IMO. I was thinking to define that, but this PR was shorter